src/styles/contact.scss

```
@@ -0,0 +1,62 @@
+.contact {
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    @include media-flex($smallWidth);
+  }
+
+  &__info {
+    display: flex;
+    justify-content: space-between;
+    width: 413px;
+    line-height: 1.7em;
+    @include content-width-small-screen;
+
+    @media (max-width: $smallWidth) {
+      flex-direction: column;
+    }
+  }
+
+  &__label {
+    display: block;
+    font-size: 14px;
+    color: $supportFontColor;
+
+  }
+
+  &__phone {
+    width: 40%;
+
+    &--number,
+    &--email {
+      font-family: $robotoFontFamily;
+      color: black;
+      text-decoration: none;
+    }
+  }
+
+  &__address {
+    width: 40%;
+  }
+
+  &__heading {
+    @include content-heading;
+    @include content-heading-small;
+  }
+
+  &__form {
+    width: 413px;
+    display: flex;
+    flex-direction: column;
+
+    @include content-width-small-screen;
+  }
+
+  &__textarea {
+    @include common-form-element(144px, #f6f8ff, black);
+    outline: none;
+    transition: background-color 1s, border 1s;

```

let's use 400ms everywhere
```suggestion
    transition: background-color 400ms, border 400ms;

```